style.scss

```diff
@@ -11,144 +12,270 @@ body {
+        font-size: 1rem;
+
+        @media (min-width: $screen-xs) {
+            font-size: 2rem;
+        }
+    }
+
+    .cards-container {
+        display: flex;
+        flex-direction: column;
+
+        @media (min-width: $screen-xs) {
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    .card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 1rem;
+        gap: 1rem;
+        background-color: $light;
+        color: $black;
+
+        @media (min-width: $screen-xs) {
+            width: 50%;
+        }
+
+        @media (min-width: $screen-md) {
+            width: 33.3333333%;
+        }
+
+        &:nth-of-type(odd) {
+            background-color: $secondary-light;
+            color: $light;
+
+            @media (min-width: $screen-xs) {
+                background-color: $light;
+                color: $black;
+            }
+
+            @media (min-width: $screen-md) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        &:nth-of-type(1),
+        &:nth-of-type(4),
+        &:nth-of-type(5) {
+            @media (min-width: $screen-xs) and (max-width: ($screen-md - 1)) {
+                background-color: $secondary-light;
+                color: $light;
+            }
+        }
+
+        .icon {
+            width: 50px;
+            height: 50px;
+        }
+
+        .title {
+            text-align: center;
+            font-weight: 400;
+            margin: 0;
+            font-size: 0.8rem;
+
+            @media (min-width: $screen-xs) {
+                font-size: 1rem;
+            }
+        }
+    }
 }
 
-.footer {
-    background-color: $primary-lighter;
+footer {
+    background-color: $primary-light;
     display: flex;
-    flex-direction: row;
     justify-content: center;
-}
 
-.footer-container {
-    display: flex;
-    flex-direction: column;
-    text-align: center;
-}
+    .footer-container {
+        display: flex;
+        flex-direction: column;
+        text-align: center;
+    }
 
-.footer-logo {
-    width: 200px;
-}
+    .footer-logo {
+        width: 50px;
+        height: 100px;
 
-.tel-footer {
-    font-size: 30px;
-    font-weight: bold;
-    margin: 15px;
-}
+        @media (min-width: $screen-xs) {
+            width: 80px;
+            height: 100px;
+        }
+
+        @media (min-width: $screen-md) {
+            width: 100px;
+            height: 100px;
+        }
+
+        @media (min-width: $screen-lg) {
+            width: 100px;
+            height: 150px;
+        }
+    }
+
+    .tel-footer {
+        font-weight: bold;
+        margin: 15px;
+        font-size: 1.5rem;
+
+        @media (min-width: $screen-xs) {
+            font-size: 1.5rem;
+        }
+
+        @media (min-width: $screen-md) {
+            font-size: 2rem;
+        }
+     }
+
+    .footer-text {
+        font-weight: bold;
+        margin: 15px;
```

Comment:
I think, there is no need to use margin for all elements in footer. Just use the same approach without marings and with `gap` as we applied in `header`.